AutomatedTesting/Gem/PythonTests/Prefab/tests/overrides/EditEntity_UnderNestedInstance.py

```diff
@@ -0,0 +1,102 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+"""
+
+def EditEntity_UnderNestedInstance():
+    """
+    Test description:
+    - Creates a "NestedPrefabs_Prefab_3" prefab containing one entity called "Test_Entity".
+    - Creates a "NestedPrefabs_Prefab_2" prefab containing "NestedPrefabs_Prefab_3" instance.
+    - Creates a "NestedPrefabs_Prefab_1" prefab containing "NestedPrefabs_Prefab_2" instance.
+    - Instantiates the "NestedPrefabs_Prefab_1" prefab twice under the level prefab.
+    - Edits the "Test_Entity" of the first "NestedPrefabs_Prefab_3" instance.
+    - Checks that after propagation, the "Test_Entity" of the first "NestedPrefabs_Prefab_3" instance contains the correct edits.
+    - Checks that after propagation, the "Test_Entity" of the second "NestedPrefabs_Prefab_3" instance does not contain any edits.
```

Comment:
nit: This is not needed imo. It's too verbose to read and follow. The below tree like hierarchy of the entities is good enough imo.